styles.css

```diff
@@ -1 +1,83 @@
-/*Maak hier jouw CSS!*/
\ No newline at end of file
+/*Maak hier jouw CSS!*/
+.grid-container {
+    display: grid;
+    background-color: #acacde;
+    row-gap: 16px;
+    column-gap: 16px;
+    grid-template-columns: 1fr;
+    grid-template-rows: 50px auto 50px 50px 50px 50px 50px;
+    grid-template-areas:
+            "header"
+            "main"
+            "aside-large"
+            "aside-small"
+            "middle"
+            "."
+            "footer";
+}
+
+
+/*Dit is voor schermen groter dan mobiel*/
```

Comment:
Alleen de properties die overschreven moeten worden, zijn opnieuw gedefinieerd in deze breakpoints.
Properties zoals display:grid, row-gap etc. worden simpelweg toegepast op alle maten tenzij we ze expliciet overschrijven in een breakpoint. Dus volgensmij waren alleen grid-template-columns, grid-template-rows en grid-template-areas hier voldoende geweest.